app/src/main/java/org/openobservatory/ooniprobe/client/OrchestrateClient.java

```diff
@@ -0,0 +1,12 @@
+package org.openobservatory.ooniprobe.client;
+
+import org.openobservatory.ooniprobe.model.api.UrlList;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface OrchestrateClient {
```

Comment:
@bassosimone how would you call this class?

Reply:
`OONIOrchestraClient`